spec/controllers/organizations_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+describe OrganizationsController do
+  let!(:organization) { create(:organization) }
+  let!(:admin) { create(:admin, is_admin: true) }
```

Comment:
Yes, I'll use a regular user here and admin in tests for `Admin::OrganizationsController`